src/controllers/UserController.ts

```diff
@@ -29,6 +30,19 @@ export class UserController {
 
     return { role: queriedRoleFromId };
   }
+
+  @Get('/:userID')
+  @ResponseSchema(AppUserProfileResponse)
+  async getUserProfile(
+    @Param('userID') userID: number,
+    @CurrentUser({ required: true }) appUser: AppUser
+  ): Promise<AppUserProfileResponse | undefined> {
+    const { id } = appUser;
```

Comment:
doesn't eslint yell at you for unused var here?

Reply:
When you say yell, do you mean it gives an error or something and doesn't run the program when doing npm run start? Also, IDK why I didn't notice this comment earlier.. I would have changed this as well. I left the ID in because it would be used in auth later so I thought might as well just leave it there.

Reply:
yeah probably just a warning then - that's fine